### ISSUE TITLE
NameError: name 'line' is not defined

### ISSUE THREAD

**Issue by zakraicik** [gretelai/gretel-synthetics#20]
This piece of code errors. I assume line should be replaced with record- is that correct?

```
for idx, record in enumerate(generate_text(config, line_validator=validate_record)):
    status = record.valid
    
    # ensure all generated records are unique
    synth_df = synth_df.drop_duplicates()
    synth_cnt = len(synth_df.index)
    if synth_cnt > records_to_generate:
        break 

    # if generated record passes validation, save it
    if status:
        print(f"({synth_cnt}/{records_to_generate} : {status})")        
        print(f"{line.text}")
        data = line.values_as_list()
        synth_df = synth_df.append({k:v for k,v in zip(df.columns, data)}, ignore_index=True)
            
```

**Comment by johntmyers** [comment]
Yea that's correct. Will update the notebook.

**Comment by zakraicik** [comment]
> Yea that's correct. Will update the notebook.

Awesome thanks!

**Comment by zredlined** [comment]
Implemented in #16 
https://github.com/gretelai/gretel-synthetics/blob/master/examples/synthetic_records.ipynb
